defaults/preferences/zotero.js

```diff
@@ -55,6 +55,7 @@ pref("extensions.zotero.groups.copyTags", true);
 pref("extensions.zotero.feeds.sortAscending", false);
 pref("extensions.zotero.feeds.defaultTTL", 1);
 pref("extensions.zotero.feeds.defaultCleanupAfter", 2);
+pref("extensions.zotero.feeds.translationTarget", false);
```

Comment:
Let's add a "last" in there for consistency: `extensions.zotero.feeds.lastTranslationTarget`